src/index.html

```diff
@@ -7,8 +7,32 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap"  rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img class="logo"
+         src="images/logo.png"
+         alt="moyo">
+      </a>
+      <nav>
+        <ul class="nav">
+          <a class="nav_link is-active" href="#">Apple</a>
+          <a class="nav_link"  href="#">Samsung</a>
+          <a class="nav_link"  href="#">Smartphones</a>
+          <a class="nav_link"
+             href="#"
+             data-qa="hover"
+          >Laptops & Computers</a>
```

Comment:
```suggestion
          >
            Laptops & Computers</a>
```
Consider fixing indentations accordingly to the code style guide